proto/src/proto/rowset.proto

```diff
@@ -35,6 +35,9 @@ message BlockIndex {
     PlainNullable = 3;
     PlainFixedChar = 4;
     PlainVarchar = 5;
+    RlePainNullable = 6;
+    RlePlainFixedChar = 7;
+    RlePlainVarchar = 8;
```

Comment:
I think `Plain` should be removed, since `Rle` and `Plain` are the same level concept? `Plain` for plain text encoding